.ci_support/linux_64_python3.12.____cpython.yaml

```diff
@@ -24,6 +24,8 @@ pin_run_as_build:
     max_pin: x.x
 python:
 - 3.12.* *_cpython
+qpdf:
+- '11'
```

Comment:
Note that re-rendering is correctly picking up `qpdf` and its pinning based on [the global pinning]( https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/c93f48525fe39073f7c0ea153af39765bb74673a/recipe/conda_build_config.yaml#L790-L791 ) to the variants here

This should now be automated going forward so maintainers shouldn't need to think about this 🙂